tests/lvm-all-devices.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '@playwright/test';
+import { IndexActor } from "../actors/index-actor";
+import { StorageActor } from "../actors/storage-actor";
+import { UserActor } from "../actors/user-actor";
+import { MainPage } from '../pages/main-page';
+import { ProductSelectionOpensusePage } from '../pages/product-selection-opensuse-page';
+import { InstallActor } from '../actors/install-actor';
+
+const minute = 60 * 1000;
+test.describe('The main page', () => {
+    test.beforeEach(async ({ page }) => {
+        const productSelectionOpensusePage = new ProductSelectionOpensusePage(page);
+        const mainPage = new MainPage(page);
+        const indexActor = new IndexActor(page, mainPage, productSelectionOpensusePage);
+        indexActor.goto();
+        indexActor.handleProductSelectionIfAny();
+    });
+
+    test('LVM with all devices', async ({ page }) => {
+        const mainPage = new MainPage(page);
+        await mainPage.expectInstallationSize();
```

Comment:
It seems on latest build we'd better do not wait installation size, see my PR https://github.com/jknphy/e2e-agama-playwright/pull/28

Reply:
Removed `expectInstallationSize()`